### PR TITLE
Update offboard docs with more details on direct actuator messages

### PR DESCRIPTION
**Pull request by qtweng**
https://github.com/PX4/PX4-Autopilot/issues/24032

### CODE REVIEW

**Comment by hamishwillee** [comment]
@beniaminopozzan Can you please sanity check this.

File: en/flight_modes/offboard.md [hunk 1]
```diff
@@ -122,6 +122,7 @@ The following offboard control modes bypass all internal PX4 control loops and s
 
 - [px4_msgs::msg::ActuatorMotors](https://github.com/PX4/PX4-Autopilot/blob/main/msg/ActuatorMotors.msg) + [px4_msgs::msg::ActuatorServos](https://github.com/PX4/PX4-Autopilot/blob/main/msg/ActuatorServos.msg)
   - You directly control the motor outputs and/or servo outputs.
+  - Currently works at lower level than mixer module. Do not publish these messages when not in offboard mode.
```
Comment (beniaminopozzan) [inline en/flight_modes/offboard.md:125]: Starting from v1.14, the control_allocator replaced the mixer module
```suggestion
  - Currently works at lower level than then `control_allocator` module. Do not publish these messages when not in offboard mode.
```

**Review by beniaminopozzan — CHANGES_REQUESTED** [review]
LGTM, just a minor naming adjustment.

**Review by hamishwillee — APPROVED** [review]
Thanks @qtweng and @beniaminopozzan

**Pull request merged**